developer/developer-faqs.md

```diff
@@ -3,6 +3,16 @@
 ## Installation
 
 ## General
+Q: What do I need to know to work with Reaction?
+
+A: To start hacking Reaction, you should probably have a basic understanding of the following technologies:
+- JavaScript, specifically ES6
+- HTML/CSS
+- Meteor experience would definitively help
+- A front-end framework/library like React
+
+Reaction tried to be as modular as possible. All user customizations are intended to live in plugins. But this flexibility comes at a cost: There are quite a few internal development APIs to know of when doing certain things like implementing a custom payment provider. It can sometimes be a bit daunting for newbies trying to do their first baby steps with Reaction. So it could absolutely be a viable first approach to just hack Reaction core directly without bothering too much with plugins, APIs, whatsoever. This makes it easier for you to get familiar with the inner workings of Reaction and leaves room for later improvement, when you're ready for the next step: Building a plugin, which rewards you with a secure upgrade path.
```

Comment:
I don't know if this really needs to be there? I think it's appropriate at the beginning of the plugin tutorial but here it seems out of place. Though I can be overruled if someone else thinks it's valuable.

Reply:
not opinionated about that. But I need to find a bridge to gap "what do I need to know" to our plugin tutorial. I'm going to shorten it.